pkg/docker/docker-entrypoint.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
+set -e
+
+# first arg is a flag.
+if [ "${1:0:1}" = '-' ]; then
```

Comment:
Is this a `bash`-ism? If so, should the shebang be `#!/bin/bash` instead?

Reply:
Done.